eve/main.yml

```diff
@@ -1553,3 +1556,277 @@ stages:
+          haltOnFailure: true
+      - ShellCommand:
+          name: Apply example-solution CRs on Bootstrap node
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f
+            /tmp/examplesolution_v1alpha1_versionserver_cr.yaml
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Verify example-solution-operator Deployment
+          env:
+            KUBECONF: "--kubeconfig /etc/kubernetes/admin.conf"
+          command: |
+            ssh -F ssh_config bootstrap --
+            bash -c '
+            ATTEMPTS=0
+            REPLICAS=$(sudo kubectl "$KUBECONF" get deployments example-solution-operator -n example-environment -o custom-columns=:status.availableReplicas)
+            until [ $REPLICAS -eq 1 ]; do
+              $REPLICAS
+              ((ATTEMPTS++))
+              if [ $ATTEMPTS -eq 10 ]; then
+                echo "Could not obtain Replicas after ${ATTEMPTS} attempts" >&2
+                exit 1
+              fi
+              sleep 3
+            done'
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Verify exampleversions Deployment
+          env:
+            KUBECONF: "--kubeconfig /etc/kubernetes/admin.conf"
+          command: |
+            ssh -F ssh_config bootstrap --
+            bash -c '
+            ATTEMPTS=0
+            REPLICAS=$(sudo kubectl "$KUBECONF" get deployments example-versionserver -n example-environment -o custom-columns=:status.availableReplicas)
+            until [ $REPLICAS -eq 3 ]; do
+              $REPLICAS
+              ((ATTEMPTS++))
+              if [ $ATTEMPTS -eq 10 ]; then
+                echo "Could not obtain Replicas after ${ATTEMPTS} attempts" >&2
+                exit 1
+              fi
+              sleep 3
+            done'
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - SetPropertyFromCommand:
+          name: Set Cluster IP as Property for example-versionserver
+          property: versionserver_cluster_ip
+          env:
+            KUBECONF: "--kubeconfig /etc/kubernetes/admin.conf"
+            SSH_CONFIG: >-
+              eve/workers/openstack-multiple-nodes/terraform/ssh_config
+          command: |
+            ssh -F $SSH_CONFIG bootstrap --
+            bash -c '
+            IP=$(sudo kubectl "$KUBECONF" get service example-versionserver -n example-environment -o custom-columns=:spec.clusterIP) &&
+            echo $IP'
+          haltOnFailure: true
+      - ShellCommand:
+          name: Verify versionserver http endpoints
+          env:
+            HTTP: "http://"
+            IP: "%(prop:versionserver_cluster_ip)s"
+            SUB_URL: ":8080/version"
```

Comment:
Same here, not sure the variable is useful, and it should not be named this way, at least have a `PORT` variable and `URL_PATH`